frontend/src/app/specification-form/specification-form.component.html

```diff
@@ -6,27 +6,43 @@
   </div>
   <div class="row top-buffer">
     <div class="col">
-      <input type="radio" name="type-radio" (click)="changeTypeRadio()" [checked]="!manualDisabled"/>
-      Upload file manually ...
+      <input type="radio" name="type-radio" (click)="changeTypeRadio()" [checked]="!remoteUploadSelected"/>
+      Upload a file from your computer
     </div>
   </div>
   <div class="row top-buffer">
     <div class="col">
       <p>
-        <input type="file" placeholder="Upload Swagger Specification" accept=".yml,.json,.yaml" ngModel
-               (change)="getFile($event)" [attr.disabled]="manualDisabled ? '' : null">
+        <input type="file" placeholder="Upload Swagger Specification" accept=".yml,.json,.yaml,.graphql" ngModel
+               (change)="onLocalFileChange($event)" [attr.disabled]="remoteUploadSelected ? '' : null">
       </p>
     </div>
+    <div class="col" *ngIf="isGraphQLFile">
+      Additionally required fields for GraphQL upload:
```

Comment:
I wouldn't write here that it is necessary only for GraphQL. Rather something like "Please add a description of the service:"